src/elements/content-uploader/progressCellRenderer.js

```diff
@@ -35,11 +35,19 @@ type Props = {
  * @param {string} [itemName]
  * @returns {FormattedMessage}
  */
-const getErrorMessage = (errorCode: ?string, itemName: ?string) => {
+const getErrorMessage = (errorCode: ?string, itemName: ?string, maxFileSize?: ?string) => {
     switch (errorCode) {
         case ERROR_CODE_UPLOAD_CHILD_FOLDER_FAILED:
             return <FormattedMessage {...messages.uploadsOneOrMoreChildFoldersFailedToUploadMessage} />;
         case ERROR_CODE_UPLOAD_FILE_SIZE_LIMIT_EXCEEDED:
+            if (maxFileSize) {
```

Comment:
I'm not sure we want to have two sources of truth (client, server) for the maximum file size. Can we add a new error code on the backend that we can key off of to ask the user to upgrade, instead?

Reply:
The plan is to get the maxFileSize from the server in the EUA (exist as CurrentUserRecord) rather than passing in a hardcoded data in the FE repo. This avoids adding new endpoint/changing current APIs for this experimental change. (reference maxFileSizeAsString in EUA)